MLOps/intro-to-ml-model-deployment-and-management/recipes/data-pumper-called-from-cli.py

```diff
@@ -0,0 +1,33 @@
+import csv
+import requests
+import time
+import datatable
+from typing import List
+from pathlib import Path
+
+# Pump data to MLOps
+
+# X is your input dt.Frame (https://github.com/h2oai/datatable)
+
+# Download hydraulic_test_data.csv from Driverless AI Datasets Page
+home = str(Path.home())
+X = dt.Frame(home + "/Downloads/hydraulic_test_data.csv")
+
```

Comment:
If someone has access to the CSV, I'm not sure it would do any good to also use `datatable` when they could read the fields directly from the CSV.  The code for this is in one of the Studio app files.

I wrote more thoughts on using (or not) this file in a previous comment.

Reply:
@orendain I referenced the studio app code for the live code py example. This script for cli could be replaced with the one from the studio app with slight modification.

Reply:
@orendain I will remove this py script for now.